### PR TITLE
Revert dark mode regressions on Windows 10

### LINKED ISSUE #7269: [Bug]: Nextcloud UI (3.14.1) is broken on Windows  10 (21H2 19044.4894) using dark mode
### ⚠️ Before submitting, please verify the following: ⚠️

- [x] This is a **bug**, not a question or a configuration issue.
- [x] This issue is **not** already reported on Github (I've searched it).
- [x] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

 Nextcloud UI (3.14.1) is completely broken in Windows 10 (21H2 19044.4894).
Image

### Steps to reproduce

Read above

### Expected behavior

Not broke.

### Which files are affected by this bug

Nextcloud Desktop

### Operating system

Windows

### Which version of the operating system you are running.

Read above

### Package

Official Linux AppImage

### Nextcloud Server version

Not sure

### Nextcloud Desktop Client version

Read above

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

Client. No need for server logs.

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by claucambra**
Closes #7269

### CODE REVIEW

**Review by mgallien — CHANGES_REQUESTED** [review]
did not work
still getting this
image
settings dialog stays light mode
image

File: src/libsync/theme.cpp [hunk 1]
```diff
@@ -62,6 +66,24 @@ bool shouldPreferSvg()
...
+    DWORDLONG const conditionMask = VerSetConditionMask(
+        VerSetConditionMask(
+            VerSetConditionMask(0, VER_MAJORVERSION, VER_GREATER_EQUAL),
+            VER_MINORVERSION, VER_GREATER_EQUAL),
+        VER_BUILDNUMBER, VER_GREATER_EQUAL);
+
+    return VerifyVersionInfo(&osvi, VER_MAJORVERSION | VER_MINORVERSION | VER_BUILDNUMBER, conditionMask) != 0;
+}
+#endif
+
```
Comment (mgallien) [inline src/libsync/theme.cpp:86]: can be replaced by Qt code doing this for every platform
see `QOperatingSystemVersion`

Comment (claucambra) [reply src/libsync/theme.cpp]: Thanks, done

File: src/gui/main.cpp [hunk 2]
```diff
@@ -82,6 +85,12 @@ int main(int argc, char **argv)
     QQuickStyle::setStyle(style);
     QQuickStyle::setFallbackStyle(QStringLiteral("Fusion"));
 
+#if defined Q_OS_WIN
+    if (QOperatingSystemVersion::current().version() <= QOperatingSystemVersion::Windows11.version()) {
```
Comment (stertingen) [inline src/gui/main.cpp:89]: This includes Windows 11, do I read this correctly?

Comment (mgallien) [reply src/gui/main.cpp:89]: yes
thanks
not intended

**Review by mgallien — APPROVED** [review]

**Pull request merged**